### PR TITLE
Account for font-weight, font-style, and others

### PR DESCRIPTION
**Pull request by drwpow**
There’s an issue where `font-weight` isn’t carried over to the ghost element, nor is `font-style`. This results in the input not being sized properly (as the plugin assumes you are always using a non-bold, non-italic font).

While I was at it, I went ahead and added some other CSS properties that affect width that don’t seem to add any obvious overhead.

### CODE REVIEW

File: autosize-input.js [hunk 1]
```diff
@@ -32,8 +32,20 @@
...
-    var elemCssText = 'font-family:' + elemStyle.fontFamily +
-                     ';font-size:' + elemStyle.fontSize
+    var elemCssText = 'letter-spacing:' + elemStyle.letterSpacing +
+                     ';font-feature-settings:' + elemStyle.fontFeatureSettings +
+                     ';font-family:' + elemStyle.fontFamily +
+                     ';font-kerning:' + elemStyle.fontKerning +
+                     ';font-size:' + elemStyle.fontSize +
+                     ';font-stretch:' + elemStyle.fontStretch +
+                     ';font-style:' + elemStyle.fontStyle +
+                     ';font-variant:' + elemStyle.fontVariant +
```
Comment (drwpow) [inline autosize-input.js]: I know it seems silly and overkill, but one ligature or alternate number in a text field could throw off the width calculation.

File: autosize-input.js [hunk 2]
```diff
@@ -32,8 +32,20 @@
     // Apply the `font-size` and `font-family` styles of `elem` on the
     // `ghost` element.
```
Comment (antonlvovych) [inline autosize-input.js]: @dangodev I think you should fix comments to be consistent with your changes

**Comment by yuanqing** [comment]
@dangodev Thanks; am OK with the change you&rsquo;re suggesting. Happy to merge it after you resolve conflicts 😃 

cc @GEEKIAM

**Comment by drwpow** [comment]
@yuanqing Cool. Blew away my file, pulled your changes, then re-added. Also adjusted the comment above.

I also modified `autosize-input.min.js`, just uglifying it with UglifyJS2 on default settings; you’re welcome to recompress that if you preferred different settings.

**Comment by yuanqing** [comment]
Just cut a `0.4.0`. By the way, you can rebuild `autosize-input.min.js` by doing `npm run dist`.

**Comment by drwpow** [comment]
Oh! Duh. I glossed right over the `package.json`. Thanks!

**Comment by Haraldson** [comment]
I’m actually struggling with this same issue right now, and I’m on 0.4.0. Help?

**Pull request merged**